app/controllers/vacancies_controller.rb

```diff
@@ -4,7 +4,10 @@ class VacanciesController < ApplicationController
   before_action :set_landing_page_description, :set_map_display, only: %i[index]
 
   def index
-    params[:location] = params[:location_facet] if params[:location_facet]
+    # Set search parameters from pretty landing page params
+    params[:location] = params[:location_facet].titleize if params[:location_facet]
```

Comment:
Do we need to titleize params[:location] as well as params[:location_facet]?

Reply:
(I don't know what the titleizing is for)

Reply:
`params[:location]` would be user input, so no. The `titleize`ing is to convert the pretty param (e.g. teaching-jobs-in-**london-town** to "London Town")